enterprise/payara.micro/src/org/netbeans/modules/fish/payara/micro/project/resources/action-mapping.xml

```diff
@@ -21,7 +21,7 @@
 -->
```

Comment:
Made the action names specific to this plugin. Some actions are dash, some are dot separated, later on it will be standardized on dots, that's how most of the actions defined in the IDE.